client/character.lua

```diff
@@ -497,6 +496,7 @@ end)
 RegisterNetEvent('qbx_core:client:playerLoggedOut', function()
     if GetInvokingResource() then return end -- Make sure this can only be triggered from the server
     chooseCharacter()
+    previewPed()
```

Comment:
Let's move the `previewPed` call inside the `chooseCharacter` function, and instead of creating the needed variables global, simply pass what is needed to `previewPed`.

Reply:
Done!